arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
```

Comment:
So should I create another json file under the new version folder or directly update the version in this file?